src/overlay/ym2151_overlay.cpp

```diff
@@ -385,32 +441,91 @@ void debugger_draw_ym_voice(int i, uint8_t *regs, ym_channel_data &ch, ym_keyon_
+		ImGui::Text("%s", "Slot");
+		if (ImGui::IsItemHovered()) {
+			ImGui::SetTooltip("Operator Slot Number");
+		}
+		ImGui::TableNextColumn();
+		ImGui::Text("%s", "DT1");
+		if (ImGui::IsItemHovered()) {
+			ImGui::SetTooltip("Detune 1\nFine pitch adjustment");
+		}
+		ImGui::TableNextColumn();
+		ImGui::Text("%s", "DT2");
+		if (ImGui::IsItemHovered()) {
+			ImGui::SetTooltip("Detune 2\nCoarse pitch adjustment");
+		}
+		ImGui::TableNextColumn();
+		ImGui::Text("%s", "MUL");
+		if (ImGui::IsItemHovered()) {
+			ImGui::SetTooltip("Frequency Multiplier\nModifies pitch by specific intervals");
+		}
+		ImGui::TableNextColumn();
+		ImGui::Text("%s", "=Freq");
+		if (ImGui::IsItemHovered()) {
+			ImGui::SetTooltip("Current frequency produced by each operator");
+		}
+		ImGui::TableNextColumn();
+		ImGui::Text("%s", "AR");
+		if (ImGui::IsItemHovered()) {
+			ImGui::SetTooltip("Attack Rate\nSpeed the volume rises from 0 to peak");
+		}
+		ImGui::TableNextColumn();
+		ImGui::Text("%s", "D1R");
+		if (ImGui::IsItemHovered()) {
+			ImGui::SetTooltip("Decay Rate 1\nSpeed the volume falls from peak to sustain level");
+		}
+		ImGui::TableNextColumn();
+		ImGui::Text("%s", "D1L");
+		if (ImGui::IsItemHovered()) {
+			ImGui::SetTooltip("Decay 1 Level (Sustain)\nVolume level at which decay rate switches to D2R");
+		}
+		ImGui::TableNextColumn();
+		ImGui::Text("%s", "D2R");
+		if (ImGui::IsItemHovered()) {
+			ImGui::SetTooltip("Decay Rate 2\nSpeed the volume decays after sustain is reached.");
+		}
+		ImGui::TableNextColumn();
+		ImGui::Text("%s", "RR");
+		if (ImGui::IsItemHovered()) {
+			ImGui::SetTooltip("Release Rate\nSpeed the volume falls to 0 when key released");
+		}
+		ImGui::TableNextColumn();
+		ImGui::Text("%s", "KS");
+		if (ImGui::IsItemHovered()) {
+			ImGui::SetTooltip("Key Scaling\nSpeed at which the envelope progresses\nEffectiveness increases with note pitch");
+		}
+		ImGui::TableNextColumn();
+		ImGui::Text("%s", "Env");
+		if (ImGui::IsItemHovered()) {
+			ImGui::SetTooltip("Current envelope state");
+		}
+		ImGui::TableNextColumn();
+		ImGui::Text("%s", "TL");
+		if (ImGui::IsItemHovered()) {
+			ImGui::SetTooltip("Total Level (volume)\nAttenuates the operator's output\n(0=loudest, 127=silent)");
+		}
+		ImGui::TableNextColumn();
+		ImGui::Text("%s", "AM");
+		if (ImGui::IsItemHovered()) {
+			ImGui::SetTooltip("Amplitude Modulation Enabled");
+		}
+		ImGui::TableNextColumn();
+		ImGui::Text("%s", "Out");
+		if (ImGui::IsItemHovered()) {
+			ImGui::SetTooltip("Signal level output by operator");
+		}
+		ImGui::TableNextRow();
 		for (int j = 0; j < 4; j++) {
-			const int     slnum  = slot_map[j] + i;
-			const uint8_t muldt1 = 0x40 + slnum;
-			const uint8_t tl     = 0x60 + slnum;
-			const uint8_t arks   = 0x80 + slnum;
-			const uint8_t d1rame = 0xA0 + slnum;
-			const uint8_t d2rdt2 = 0xC0 + slnum;
-			const uint8_t rrd1l  = 0xE0 + slnum;
-			auto &        slot   = ch.slot[j];
+			const int     slnum     = slot_map[j] + i;
```

Comment:
I'm not sure why these lines had extra whitespace padded into them. I anticipate that this whitespace will get clobbered in a subsequent change to this file.

A note on style: I'm relying on clang-format for newlines and whitespace-related style. I know that some of the options tend to create seemingly needless line-edits (such as another case above where the whitespace aligning `=` signs changed, because of neighboring lines that were added and changed the evaluated alignment. (And a note on that note: If you don't want a group of assignment to align with another group, for whatever reason, I would actually prefer an empty line separating them.)

There aren't enough commits for it to _really_ matter at this point, so I'm not worried about it. Just notes for the future.